BasicSample/Assets/Rendering/Scripts/RenderingSettings.cs

```diff
@@ -66,6 +89,23 @@ void Update()
                 {
                     m_statusPanel.text += "\tTarget reprojection mode not supported!\n";
                 }
+
+            }
+
+            if (Camera.allCameras.Length > 0)
+            {
+                Vector3 rightEye = (Vector3)(Camera.allCameras[0].GetStereoViewMatrix(Camera.StereoscopicEye.Right).inverse.GetColumn(3));
```

Comment:
I understand this is what you are doing in internal tests to workaround Unity's bug on StereoSeparation property.  I don't think this is the right thing to do, since this setup assumes the Camera is stereo and only contains 2 views.   Which is not always true.

Two options:  A: Use unity's StereoSeparation property instead, even if it's displaying the wrong value today.  We can put a comment in code to indicate there might be bugs waiting for future unity fixes, but it's better than working around logic.
B: Use "Tracked Pose Driver" with eye poses setup which also gives left and right eye poses.  https://docs.unity3d.com/2018.3/Documentation/ScriptReference/SpatialTracking.TrackedPoseDriver.TrackedPose.html

Reply:
@yl-msft just a note, i chose option A here since the api in option B does not seem to be available in Unity 2020 and higher. Do you think its worth using this instead: https://docs.unity3d.com/ScriptReference/XR.Eyes.html because i am not sure how long it will take unity to fix the bug.

Reply:
It's a good choice for our internal testing code, but we don't have to workaround for unity's bug on public samples.  It's good to be here to let unity fix their side of the bug.  Let's keep eyes on the fix and update this comment accordingly.